plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/BaseElasticsearchConnectorTest.java

```diff
@@ -1003,6 +1036,9 @@ public void testNestedVariants()
         assertQuery(
                 "SELECT a.b.c FROM nested_variants",
                 "VALUES 'value1', 'value2', 'value3', 'value4'");
+
+        assertThatThrownBy(() -> computeActual("SELECT a.\"b.c\" FROM nested_variants"))
+                .hasMessageContaining("Column 'a.b.c' cannot be resolved");
```

Comment:
Please fix the error message in `Add test for column name containing special characters in Elasticsearch` commit itself.

```suggestion
        assertThat(query("SELECT a.\"b.c\" FROM nested_variants"))
                .failure()
                .hasErrorCode(INVALID_COLUMN_REFERENCE)
                .hasMessageContaining("Column reference 'a.b.c' is invalid");
```